build/build.sh

```diff
@@ -35,7 +35,7 @@ fi
 LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildStatus=${GITSTATUS}"
 
 : ${GITTAG:=$(git describe 2> /dev/null || echo "unknown")}
-LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildTag=${GITTAG}"
+LD_EXTRAFLAGS+=" -X ${REPO_PATH}/pkg/version.buildTag=${GITTAG} -X ${REPO_PATH}/pkg/version.minimumSupportedVersion=${MINIMUM_SUPPORTED_VERSION}"
```

Comment:
Please initialize this variable to a reasonable default value if not provided externally, since not all builds are done via RPM build.

Reply:
This is a copy of the 2.0 codebase which is already working for product builds

Reply:
Note the additional code does default to all versions if this is not configured

Reply:
I've confirmed that 'make compile' works without issue

Reply:
yeah, it compiles fine

Reply:
We will have to add a default value anyway in the future (even if empty), when working on MAISTRA-2107.

Reply:
Sounds good